src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -58,4 +59,18 @@ protected static function createClient(array $options = [], array $server = [])
 
         return self::getClient($client);
     }
+
+    /**
+     * @throws \Exception
+     */
+    public static function assertThat($value, $constraint, $message = ''): void
```

Comment:
Not sure if I should type `$onstraint` and `$message`, for BC reasons, because they are typed starting from PHPUnit 7.